### PR TITLE
Update filters-device-properties.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note on devicetrusttype 

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4304

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/filters-device-properties.md [hunk 1]
```diff
@@ -207,6 +207,9 @@ You can use the following device properties in your managed device filter rules:
   - Windows 11
   - Windows 10
 
+  > [!NOTE]
+  > **`deviceTrustType`** is a shared property between Entra ID and Intune, the allowed values under the expression editor are actually different for Entra ID groups and device filters.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/filters-device-properties.md:211]:
```suggestion
  > **`deviceTrustType`** is a shared property between Entra ID and Intune; the allowed values under the expression editor are different for Entra ID groups and device filters.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MandiOhlinger Please review and signoff this PR. Thanks.

**Review by MandiOhlinger — APPROVED** [review]

**Pull request merged**